### PR TITLE
[CARBONDATA-1326] Fixed findbug issue and univocity-parsers jar version to 2.2.1

### PR DESCRIPTION
**Pull request by pawanmalwal**
Updated univocity-parsers jar version from 1.5.6 to 2.2.1 in processing/pom.xml and fixed one findbug issue.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: core/src/main/java/org/apache/carbondata/core/scan/filter/FilterExpressionProcessor.java [hunk 1]
```diff
@@ -163,17 +163,17 @@ public FilterResolverIntf getFilterResolver(Expression expressionTree,
   }
 
   private String joinByteArray(byte[] bytes, char delimiter) {
-    StringBuffer byteArrayAsString = new StringBuffer();
-    byteArrayAsString.append("");
+    String byteArrayAsString = "";
```
Comment (sraghunandan) [inline core/src/main/java/org/apache/carbondata/core/scan/filter/FilterExpressionProcessor.java]: do not change this to String,Findbugs will give error.
though java optimises internally when strings are concatenated, findbugs doesn't recognise that

Comment (pawanmalwal) [inline core/src/main/java/org/apache/carbondata/core/scan/filter/FilterExpressionProcessor.java]: Changed to StringBuffer per your suggestion.

**Comment by sraghunandan** [comment]
Master already has stringbuffer implementation

**Comment by pawanmalwal** [comment]
Yes thats right. This PR also has some code change to avoid 1 more string object creation in the same method and another chanage to update univocity-parsers jar version to 2.2.1 in processing/pom.xml. Please review.

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by sraghunandan** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by pawanmalwal** [comment]
retest this please

**Comment by pawanmalwal** [comment]
retest this please

**Comment by pawanmalwal** [comment]
Duplicate PR - univocity jar changes done in PR https://github.com/apache/carbondata/pull/1532

**Pull request closed without merging**